cocos/ui/UIEditBox/UIEditBoxImpl-mac.mm

```diff
@@ -196,10 +196,10 @@ of this software and associated documentation files (the "Software"), to deal
 void EditBoxImplMac::updateNativeFrame(const cocos2d::Rect &rect)
 {
     GLView* eglView = Director::getInstance()->getOpenGLView();
-    auto viewPortRect = eglView->getViewPortRect();
+    auto frameSize = eglView->getFrameSize();
```

Comment:
As I know, in`FIXED_WIDTH` mode `viewport height` and `frame height` is expected to be the same.

Reply:
you can set:

Reply:
you can set:
   setFrameSize(900, 500)
   setDesignSize(800, 500, NO_BORDER)

here is my app with wrong position:
<img width="572" alt="1" src="https://user-images.githubusercontent.com/2108342/69476069-04bd4580-0e10-11ea-9619-4486386d9abb.png">
<img width="417" alt="2" src="https://user-images.githubusercontent.com/2108342/69476070-04bd4580-0e10-11ea-800c-d4ff824786fb.png">